paychmgr/manager.go

```diff
@@ -344,6 +344,7 @@ func (pm *Manager) trackInboundChannel(ctx context.Context, ch address.Address)
 	return pm.store.TrackChannel(stateCi)
 }
 
+// TODO: secret vs proof doesn't make sense, there is only one, not two
```

Comment:
I believe we had both at some point in the past

Reply:
We did. But it wasn't usable so we removed it from actors.